security/authenticator_manager.rst

```diff
@@ -616,7 +616,7 @@ would initialize the passport like this::
                 return $passport;
             }
 
-            public function createAuthenticatedToken(PassportInterface $passport, string $firewallName): TokenInterface
+            public function createAuthenticatedToken(Passport $passport, string $firewallName): TokenInterface
```

Comment:
The method should also be renamed, right?

Reply:
I renamed while merging ... but I have a question. The new code is:

```php
public function createAuthenticatedToken(PassportInterface $passport, string $firewallName): TokenInterface
{
    trigger_deprecation('symfony/ldap', '5.4', 'Method "%s()" is deprecated, use "%s::createToken()" instead.', __METHOD__, __CLASS__);

    return $this->createToken($passport, $firewallName);
}

public function createToken(PassportInterface $passport, string $firewallName): TokenInterface
{
    // @deprecated since Symfony 5.4, in 6.0 change to:
    // return $this->authenticator->createToken($passport, $firewallName);
    return method_exists($this->authenticator, 'createToken')
        ? $this->authenticator->createToken($passport, $firewallName)
        : $this->authenticator->createAuthenticatedToken($passport, $firewallName);
}
```

So, the method is `createToken(PassportInterface $passport, ...)` but in the docs we use `createToken(Passport $passport, ...)` Do you see a problem here? Thanks!

Reply:
That is a typo in the LDAP authenticator indeed. Other authenticators (including the interface) use `createToken(Passport $passport, ...)`